INeedCoffee/CoffeeShop/ClosestCoffeeShopsFinder.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace INeedCoffee
+{
+    public static class ClosestCoffeeShopsFinder
+    {
+        public static void Find(List<CoffeeShop> coffeeShopsList, int numberOfClosestCoffeeShopsToShow, int numberOfDecimalsToRoundDistanceTo)
```

Comment:
Does not look too OOP. We are overusing static methods
https://stackoverflow.com/questions/2671496/java-when-to-use-static-methods

Reply:
I had a moment in which I used the Visual Studio’s fix all utility without focusing on each method to define it properly